packages/mdl-textfield/mdl-textfield.scss

```diff
@@ -0,0 +1,356 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/variables";
+@import "mdl-theme/variables";
+@import "mdl-theme/mixins";
+
+$mdl-textfield-error-on-light: #d50000;
```

Comment:
Not at the moment. If the community asks for it we could consider adding it in but the spec is relatively clear in the mocks about the specific colors it wants used for these messages. I'd rather limit the API scope for now and then make associative changes if people request it.